braindecode/datautil/serialization.py

```diff
@@ -62,42 +64,63 @@ def save_concat_dataset(path, concat_dataset, overwrite=False):
 
 
 def load_concat_dataset(path, preload, ids_to_load=None, target_name=None):
-    """Load a stored BaseConcatDataset of BaseDatasets or WindowsDatasets from
+    """Load stored BaseConcatDataset(s) of BaseDatasets or WindowsDatasets from
     files
 
     Parameters
     ----------
     path: str
-        path to the directory of the .fif and .json files
+        path to the (enumerated subdirectories of the) directory of the .fif
+        and .json files
     preload: bool
         whether to preload the data
     ids_to_load: None | list(int)
-        ids of specific signals to load
+        ids of specific datasets to load
     target_name: None or str
         Load specific column as target. If not given, take saved target name.
 
     Returns
     -------
     concat_dataset: BaseConcatDataset of BaseDatasets or WindowsDatasets
     """
-    assert ((os.path.isfile(os.path.join(path, '0-raw.fif')) +
-             os.path.isfile(os.path.join(path, '0-epo.fif'))) == 1), (
-        "Expect either raw or epo to exist inside the directory")
+    # assume we have a single concat dataset to load
     concat_of_raws = os.path.isfile(os.path.join(path, '0-raw.fif'))
+    concat_of_epochs = os.path.isfile(os.path.join(path, '0-epo.fif'))
+    paths = [path]
+    # assume we have multiple concat datasets to load
+    if not (concat_of_raws + concat_of_epochs):
```

Comment:
```suggestion
    if not (concat_of_raws or concat_of_epochs):
```

seems better as it is a logical comparison